__tests__/FSStorage.test.js

```diff
@@ -0,0 +1,94 @@
+import {
+    getCacheFromPhone,
+    setModificationsCacheFromPhone
+} from '../lib/FSStorage';
+import FsLogger from '../lib/FsLogger';
+import AsyncStorage from '@react-native-community/async-storage';
+
+/// Data Mock , inspired from real case
+export const dataMock = {
+    fsModifications: [
+        {
+            id: 'bqjhehmirtfg026713ig',
+            variationGroupId: 'bqjhehmirtfg026713jg',
+            variation: {
+                id: 'bqjhehmirtfg026713kg',
+                modifications: {
+                    type: 'JSON',
+                    value: {
+                        RNBackGroundColor: 'pink',
+                        RN_btn_title: 'Save blog',
+                        RN_label_content: 'Content of post',
+                        RN_label_title: 'Title of post',
+                        btnTitle: 'Shop'
+                    }
+                },
+                reference: false
+            }
+        }
+    ],
+    config: {
+        fetchNow: true,
+        activateNow: false,
+        enableConsoleLogs: true,
+        enableErrorLayout: false,
+        enableSafeMode: false,
+        nodeEnv: 'production',
+        flagshipApi: 'https://decision-api.flagship.io/v1/',
+        apiKey: null,
+        initialModifications: null
+    }
+};
+
+export const emptyDataMock = {
+    fsModifications: []
+};
+
+it('checks getCacheFromPhone', async () => {
+    setModificationsCacheFromPhone(dataMock, FsLogger.getLogger());
+
+    return getCacheFromPhone(FsLogger.getLogger()).then((data) => {
+        expect(AsyncStorage.getItem).toBeCalledWith('@storage_fsModifications');
+
+        /// Check if is array
+        expect.arrayContaining(data);
+
+        // /// Read first object
+        const fsModif = data['modifications'][0]; /// Hard access , because we know the mock response don't have an empty array
+
+        console.log(fsModif);
+
+        // /// VariationGroupId
```

Comment:
Too many slashes in your JS commentary

`// /// VariationGroupId`

should be:

`// VariationGroupId`

Reply:
Correct every line with this issue please

Reply:
Done